src/main/java/org/fundacionjala/at15/katas/langtonant/daniel/Ant.java

```diff
@@ -0,0 +1,137 @@
+package org.fundacionjala.at15.katas.daniel.langtonant;
+
+public class Ant{
+    private int posX;
+    private int posY;
+    private Grid grid;
+    private String direction;
+    private boolean finished;
+
+    public Ant(int x, int y, Grid grid, String direction){
+        posX = x;
+        posY = y;
+        this.grid = grid;
+        this.direction = direction;
+        finished = false;           // the game is ready
+    }
+    // 0 -> negro   (left)
+    // 1 -> blanco (right)
+    public void nextStep(){
+        switch (direction) {
+            case "up":
+                if(grid.getSquare(posX, posY) == 1){    
+                    if(isValidNextPosition(posX, posY+1, "right")){
```

Comment:
Try to keep consitency on the spacing before and after operators. For example, in this line, we have no spaces before and after `+` operator in `posY+1`, but line 25 adds spaces before and after `+=` operator. In this case, it would be recommendable to add spaces before and after `+` operator. Same for the rest of the code